suse_openstack_cloud

```diff
@@ -101,11 +101,16 @@ web_ui_admin_default_password "s3kr1t"
 .... "bmc_user"=>"root", "bmc_password"=>"cr0wBar!"}, {"token"=>"l33t", ...
 neutron_admin_password=s3kr1t
 rabbit_password=s3kr1t
-connection=postgresql://nova:s3kr1t@192.168.124.81/nova
+connection = mysql+pymysql://nova_api:s3kr1t@192.168.124.81/nova
    password: p4ssw0rt
    service_password: service_p4ssw0rt
    stack_domain_admin_password: 4dm1n_p4ssw0rt
    auth_encryption_key: 4u7h_ke7
+admin_password = secrete
+password = secrete
```

Comment:
Your spelling of `secret` is somewhat unconventional unless you meant something else :smile:.

Reply:
Yeah, I'm slightly puzzled by that too :-)

Reply:
http://codesearch.openstack.org/?q=secrete&i=nope&files=&repos=

Reply:
OK thanks for the info - that's definitely non-obvious enough to merit a mention in a comment or the commit message.

Reply:
Interesting. So this creative new way of spelling is catching on! :smile: I agree with @aspiers, a note in the commit message would help.

Reply:
Fixed